backend/src/schedule_meeting.rs

```diff
@@ -1,3 +1,4 @@
+use crate::uuid::gen_uuid_with_xorshift;
```

Comment:
We do not like use statement, use the full crate path where you are calling this function. This facilitates code review as when I see a function getting called, I do not have to wonder from which crate is this function coming. When reading I do not want to jump. It's like register vs ram. Registers are faster than main memory. The line I am seeing right now is like register, it's fast, it's right in front of me, but the line near the top of the file is very far. So if you used full path I would not have to jump around in file to read a single line.